tracer/src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -332,5 +329,12 @@ public bool Equals(Key other)
                        TagPrefix == other.TagPrefix;
             }
         }
+
+        private static class DelegateCache<THeaders>
+            where THeaders : IHeadersCollection
+        {
+            public static readonly Func<THeaders, string, IEnumerable<string?>> Getter = (headers, name) => headers.GetValues(name);
+            public static readonly Action<THeaders, string, string?> Setter = (headers, name, value) => headers.Set(name, value);
```

Comment:
```suggestion
            public static readonly Func<THeaders, string, IEnumerable<string?>> Getter = static (headers, name) => headers.GetValues(name);
            public static readonly Action<THeaders, string, string?> Setter = static (headers, name, value) => headers.Set(name, value);
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.